### PR TITLE
MAINT Fix broken sphinx-book-theme reference in CI

### PR DESCRIPTION
**Pull request by ArturoAmorQ**
The CI was running on the branch related the PR https://github.com/executablebooks/sphinx-book-theme/pull/754. When the branch was erased by the merge, the CI got broken.

This PR fixes the issue by pointing to the respective commit.

### CODE REVIEW

File: requirements-dev.txt [hunk 1]
```diff
@@ -6,7 +6,7 @@ plotly
 jupyter-book>=0.11
 # Partial fix for the navbar scrollToActive behavior:
 # https://github.com/executablebooks/sphinx-book-theme/issues/541
-sphinx-book-theme @ git+https://github.com/ogrisel/sphinx-book-theme@fix-bd-docs-nav
+sphinx-book-theme @ git+https://github.com/executablebooks/sphinx-book-theme/commit/aca0f7fd39314ec3283f1ff798b4bf8ee9b49cad
```
Comment (ogrisel) [inline requirements-dev.txt:9]: git+https://github.com/executablebooks/sphinx-book-theme@aca0f7fd39314ec3283f1ff798b4bf8ee9b49cad

maybe?

Comment (ArturoAmorQ) [inline requirements-dev.txt:9]:
```suggestion
sphinx-book-theme @ git+executablebooks/sphinx-book-theme@aca0f7fd39314ec3283f1ff798b4bf8ee9b49cad
```

File: requirements-dev.txt [hunk 2]
```diff
@@ -6,7 +6,7 @@ plotly
 jupyter-book>=0.11
 # Partial fix for the navbar scrollToActive behavior:
 # https://github.com/executablebooks/sphinx-book-theme/issues/541
-sphinx-book-theme @ git+https://github.com/ogrisel/sphinx-book-theme@fix-bd-docs-nav
+sphinx-book-theme @ git+executablebooks/sphinx-book-theme@aca0f7fd39314ec3283f1ff798b4bf8ee9b49cad
```
Comment (ogrisel) [inline requirements-dev.txt:9]:
```suggestion
sphinx-book-theme @ git+https://github.com/executablebooks/sphinx-book-theme.git@aca0f7fd39314ec3283f1ff798b4bf8ee9b49cad
```

**Review by ogrisel — APPROVED** [review]

**Comment by ArturoAmorQ** [comment]
I think we still have a broken CI: Later PRs are rising:
```python-traceback
error: subprocess-exited-with-error

× Building wheel for sphinx-book-theme (pyproject.toml) did not run successfully.
│ exit code: 1
```

**Pull request merged**